index.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
```

Comment:
Why not @koajs?

Reply:
Most tutorials are in express, as this is my first contact with Node, I'm thinking of using express.
Is there a big difference between express and koa?

Reply:
Koajs is from the same creators of express, so its kind of the "spiritual" sucessor. It has overall more features out of the box.

Reply:
Nop, because @koajs has made by Express developers, so the DX is similar. You have some differences, but it isn't so much to cause awkwardness.

Reply:
I understand, I believe that because of the content on the internet I will continue with Express!

Thanks for the discussion guys